### ISSUE TITLE
Installer 3.0.3 erased my Windows PATH variable.

### ISSUE THREAD

**Issue by exjam** [devkitPro/installer#15]
It actually erased my PATH variable. Luckily I decided to check it after the warning prompt came up telling me it attempted to erase it - apparently it succeeded too.

For future reference: you can recover path variable using process explorer to see the environment variables applications are running under, be careful to erase the applications folder from the start of the PATH variable and the user paths from the end...

**Comment by exjam** [comment]
Also - setting PATH should be an optional part of the installer. I imagine most people do not want msys2 in their PATH.

**Comment by splapoon** [comment]
you should close this issue

**Comment by GalaxyGaming2000** [comment]
yep it did stuff to my path variable too

**Comment by Yoti** [comment]
Yeah, the fuck isn't fixed yet. It puts their dirs above of all (even system) ones and break, for example, `timeout` command.

**Comment by PLMMJ-FH** [comment]
On Windows 11, didn't get a popup or anything but unfortunately didn't check issues until after it was probably too late. I also don't have process explorer. What to do?
